wherehows-web/app/components/visualization/charts/horizontal-bar-chart.ts

```diff
@@ -123,6 +153,16 @@ export default class HorizontalBarChart extends Component {
     };
   }
 
+  constructor() {
+    super();
```

Comment:
`super(...arguments);`

Reply:
Ah interesting. That used to throw a typescript error for me but I guess after the update it works haha.